src/templates/admin/elements/issue/table_of_contents.html

```diff
@@ -41,13 +41,13 @@ <h2>Table of Contents</h2>
                                 <td>
                                     <a href="{% url 'manage_archive_article' article.pk %}">{{ article.title|safe }}</a>
                                 </td>
-                                <td>{{ article.date_submitted }}</td>
                                 <td>{{ article.date_published }}</td>
                                 <td>{{ article.author_list }}</td>
                                 <td>{{ article.section.name }}</td>
                                 <td>{{ article.get_doi }}</td>
                                 <td>{{ article.get_pubid }}</td>
-                                <td>{{ article.page_range }}</td>
+                                <td>{{ article.first_page|default_if_none:"--" }}</td>
+                                <td>{{ article.page_range|default_if_none:"--"  }}</td>
```

Comment:
Small thing--can we just make this `article.page_numbers` since that's the name of the field in the table head? `page_range` is different--it's a calculated field that could potentially draw on `first_page` and `last_page`, keeping folks from effectively testing changes to the `page_numbers` field.

Reply:
There are too many page fields >.<